removefiles/activity.go

```diff
@@ -14,17 +14,21 @@ import (
 
 const ActivityName = "remove-files-activity"
 
-type Activity struct{}
+type Config struct {
+	// RemoveNames is the comma separated list of filenames that should be
+	// removed from all transfers.
+	RemoveNames string
+}
 
-func NewActivity() *Activity {
-	return &Activity{}
+type Activity struct {
+	cfg Config
 }
```

Comment:
I guess the usage of a `Config` struct could be useful if we add more fields to it or if the `Activity` struct has multiple fields. Considering the current implementation, I'd just put the `RemoveNames` string inside the `Activity` struct.

Reply:
My intention is to import `removefiles.Config` into the Preprocessing MoMA config struct: https://github.com/artefactual-sdps/preprocessing-moma/blob/main/internal/config/config.go#L14, which matches the pattern used for the other config values.  I think this will work well with including `removefiles` in a modular way, as the configuration will be added under a separate heading in the preprocessing-moma config file.

Since I will have the `config.Config` struct anyway, I think it makes sense just to include it as is in the Activity struct.